index.html

```diff
@@ -0,0 +1,78 @@
+<fieldset>
+<form id="survey-form">
+<label for="name" id="name-label">Name:</label>
+<input type="text" id="name" placeholder="Enter your name" required>
+<br>
+<label for="email" id="email-label">Email:</label>
+<input type="email" id="email" placeholder="Enter your email" required>
+<br>
+<label for="number" id="number-label">Number:</label>
+<input type="number" id="number" placeholder="Enter a number between 1 and 10" min="1" max="10" required>
+<br>
+<hr>
+<label for="dropdown">Which option best describes your current role? </label>
+<select id="dropdown">
+<option hidden value="option1">select current role</option>
+<option value="option2">Student</option>
+<option value="option3">full time job</option>
+<option value="option4">full time learner</option>
+<option value="option5">prefer not to say</option>
+<option value="option6">other</option>
+</select>
+<br>
+<hr>
+<label>Would you recommend free code camp for a friend ?</label><br>
+<input type="radio" id="option1" name="options" value="option1">
+<label for="option1">Definitely</label><br>
+<input type="radio" id="option2" name="options" value="option2">
+<label for="option2">Maybe</label><br>
+<input type="radio" id="option1" name="options" value="option1">
+<label for="option3">Not sure</label><br>
+
+
+
+
+<hr>
+<label>What would you like to see improved ?</label><br>
+<input type="checkbox" id="checkbox1" name="checkbox" value="option1">
+<label for="checkbox1">Additional courses</label><br>
+<input type="checkbox" id="checkbox1" name="checkbox" value="option1">
+<label for="checkbox1">Front-end projects</label><br>
+<input type="checkbox" id="checkbox2" name="checkbox" value="option2">
+<label for="checkbox2">Back-end projects</label><br>
+<input type="checkbox" id="checkbox3" name="checkbox" value="option3">
+<label for="checkbox3">Data visualization</label><br>
+<input type="checkbox" id="checkbox4" name="checkbox" value="option4">
+<label for="checkbox4">Forum</label><br>
+<input type="checkbox" id="checkbox5" name="checkbox" value="option5">
+<label for="checkbox5">Challenges</label><br>
+<input type="checkbox" id="checkbox6" name="checkbox" value="option6">
+<label for="checkbox6">Opensource community</label><br>
+<input type="checkbox" id="checkbox7" name="checkbox" value="option7">
+<label for="checkbox7">Gitter help rooms</label><br>
+<input type="checkbox" id="checkbox8" name="checkbox" value="option8">
+<label for="checkbox8">Videos</label><br>
+<input type="checkbox" id="checkbox9" name="checkbox" value="option9">
+<label for="checkbox9">City meetups</label><br>
+<input type="checkbox" id="checkbox10" name="checkbox" value="option10">
+<label for="checkbox10">wiki </label><br>
+<br>
+<hr>
+<label for="comments">Any comments or suggestion</label><br>
+<textarea placeholder="Enter your comment here ..."id="comments"></textarea>
+<br><hr>
+<button type="submit" id="submit">Submit</button>
+</form>
+</fieldset>
+</body>
+</html>
```

Comment:
please use the right git hub flow. to protect the main project from being changed everytime you make changes to your project  .use "git checkout -b "branch name" to create new branch .

Reply:
Good job commenting on Github flow  and giving a good reason why it is needed 🚀. 
However, as mentioned earlier, in-line comments are meant for lines or block suggestions. Review suggestions related to Github flow can be placed inside your Review summary. 📜 

- [ ] Please make sure other none code related suggestions are in your summary.